action-server/covidflow/utils/geocoding.py

```diff
@@ -19,20 +21,26 @@ def __init__(self):
         key = os.environ[GOOGLE_API_KEY_ENV]
         self.client = googlemaps.Client(key=key)
 
-    def get(self, address: str) -> Optional[Tuple[float, float]]:
-        try:
+    def get(
```

Comment:
instead of 2 parameters that can be none at the same timen, we could have `for_postal_code` and `for_address` methods